glue/configure_includes/configure_linux_arm.mk

```diff
@@ -27,36 +27,54 @@ else
   TEMP_TARGET_DATASIZE:=32
 endif
 
+ifeq ($(OPENJ9_DOCKER_CC_ARM),1)
+  TMP_HOST = $$(OPENJ9_CC_PREFIX)
+else
+  TMP_HOST = arm-bcm2708hardfp-linux-gnueabi
+endif
+
 ifeq (linux_arm, $(SPEC))
```

Comment:
This looks like the wrong test, it should be
ifneq (,$(findstring linux_arm,$(SPEC)))
to also accept linux_arm_docker.